src/corona_hakab_model/common/state_machine.py

```diff
@@ -90,6 +90,13 @@ def transfer(self, agents: Set[Agent], origin_state: State) -> Iterable[PendingT
             agent_dest = dests[agent_age][current_dest_index] # Destination of current agent
             current_duration_index = age_dest_duration_index[(agent_age, agent_dest)]
             time_to_next_state = agents_durations[(agent.age, agent_dest)][current_duration_index]
+            if type(time_to_next_state) is tuple:
```

Comment:
isinstance

Reply:
this line was removed in last commit (it belongs to a deprecated feature I thought would be useful)